tests/cases/fourslash/findAllRefsObjectLiteral.ts

```diff
@@ -0,0 +1,19 @@
+/// <reference path="fourslash.ts" />
+
+// @noLib: true
+
+////interface I { [|{| "isDefinition": true |}x|]: number; }
+////function f(b: boolean) {
+////    const o = b ? { [|{| "isWriteAccess": true, "isDefinition": true |}x|]: 0 } : { [|{| "isWriteAccess": true, "isDefinition": true |}x|]: 1 };
```

Comment:
Why should this match to interface `I`? Where does it indicate that it needs to match `I`

Reply:
The type is an exact match since they're both `x: number`. We don't know that this is intended to have the same meaning as `I`, but we don't know that it doesn't, and the probability is good since they have the same property names with the same types. The alternative is not find any references but the current location (and `o.x`, but that only works for the first object literal here due to union subtype elimination), and I figured something would be better than nothing in this case.

Reply:
I don't think we should match I for these object literals. This could potentially match unintended interfaces and that could be bad. @DanielRosenwasser what are your thoughts on this.